PowerValidatedSolutions.psm1

```diff
@@ -32530,18 +32591,47 @@ Function Test-vSphereApiAuthentication {
 Export-ModuleMember -Function Test-vSphereApiAuthentication
 
 Function Test-NSXTConnection {
+    <#
+        .SYNOPSIS
+        Check network connectivity to an NSX Manager.
+
+        .DESCRIPTION
+        Checks the network connectivity to an NSX Manager.
+        Supports testing a connection on ports 443 (HTTPS) and 22 (SSH). Default: 443 (HTTPS).
+
+        .EXAMPLE
+        Test-NSXTConnection -server sfo-m01-vcf01.sfo.rainpole.io
```

Comment:
This should also change to nsx.

Reply:
```suggestion
        Test-NSXTConnection -server sfo-m01-nsx01.sfo.rainpole.io
```

Reply:
Addressed in 14b2e73bdb02f296b8ecbcda1032260e254946ce.